client-ui/src/app/components/lineage/lineage-details/operation-properties-details/operation-properties-details.component.ts

```diff
@@ -71,14 +72,22 @@ export class OperationPropertiesDetailsComponent implements AfterViewInit, OnDes
             if (!PropertiesComponents.has(name)) {
               name = OperationType.Generic
             }
-            const properties = this.getProperties(store.detailsInfos, store.attributes)
-            const component = type == OperationType.Write ? PropertiesComponents.get(OperationType.Write) : PropertiesComponents.get(name)
-            const factory = this.componentFactoryResolver.resolveComponentFactory(component)
-            const instance = container.createComponent(factory).instance
-            instance.properties = properties
-            instance.propertyName = name
-            instance.propertyType = type
-            if (!this.changedetectorRef['destroyed']) this.changedetectorRef.detectChanges()
+            let properties: Property[] = []
+            let component: Type<PropertiesComponent>
+            try {
+              properties = this.getProperties(store.detailsInfos, store.attributes)
+              component = type == OperationType.Write ? PropertiesComponents.get(OperationType.Write) : PropertiesComponents.get(name)
+            } catch (error) {
+              console.error(error)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Calls to 'console.error' are not allowed.](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4600414)